aten/src/ATen/native/cuda/BinaryRemainderKernel.cu

```diff
@@ -20,6 +26,18 @@ void remainder_kernel_cuda(TensorIterator& iter) {
         return r;
       });
     });
+  }
+  else if (isComplexType(iter.dtype())) {
+    AT_DISPATCH_COMPLEX_TYPES(iter.dtype(), "remainder_cuda", [&]() {
+      gpu_kernel_with_scalars(iter,
+        []GPU_LAMBDA(scalar_t a, scalar_t b) -> scalar_t {
+          scalar_t mod = a - floor_wrapper (a / b) * b;
+          const scalar_t zero = scalar_t(0); 
+          if ((mod != zero) && ((std::real(b) < std::real(zero)) != (std::real(mod) < std::real(zero)))) 
+            mod += b;
+          return mod;
+        });
```

Comment:
I am not sure this is the right way to implement `floor` for complex. I would go for
```
floor(z)  = ceil(|z|) * z/|z|.
```